src/edifact.ts

```diff
@@ -1067,25 +1067,34 @@ export class MessageDetails implements Segment {
 
 // DTM
 
-export class DateTimePeriod implements Segment {
-
-    tag = "DTM";
+class DateTimePeriod {
```

Comment:
This name clashes with the one of the `DateTimePeriod` segment. 

```suggestion
class DateTimePeriodData {
```

to be in inline with the scheme used in the `edifact.ts` file